build.gradle.kts

```diff
@@ -136,7 +136,9 @@ dependencies {
     because("Graphs")
   }
   implementation(libs.relocated.antlr)
-  implementation(libs.relocated.asm)
+  //implementation(libs.relocated.asm)
```

Comment:
Left comments in because it's WIP -- You may have reasons I don't understand for using the shadow jar so didn't want to go too far down this route.

Reply:
Yup, as I said on the issue, I do have reasons to use the shadow jar.

Reply:
Sounds good. I may need some help with the final step since it seems like it's two parts (publish new shaded jar ***then*** test new version). In the mean time I'll revert the changes I made here. I can use the bad one for now since I'm trying to get a failing test case first.